### PR TITLE
(maint) Updates pdk to 1.9.0.

### PR DESCRIPTION
**Pull request by tavilarion**
This is required in order to keep in line with our release process.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,9 +26,6 @@ matrix:
     -
       env: PUPPET_GEM_VERSION="~> 5.0" CHECK=parallel_spec
       rvm: 2.4.4
-    -
-      env: PUPPET_GEM_VERSION="~> 4.0" CHECK=parallel_spec RUBYGEMS_VERSION=2.7.8 BUNDLER_VERSION=1.17.3
```
Comment (melissa) [inline .travis.yml:33]: Same question as with cron, do we want to drop testing against puppet4?

**Review by melissa — COMMENTED** [review]
ping @puppetlabs/platform-os for review

File: .travis.yml (hunk 1 above)
Comment (branan) [reply .travis.yml:33]: I think it's fine (duplicating here for completeness)

File: metadata.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "puppetlabs-selinux_core",
   "version": "1.0.1",
-  "author": "Puppet Labs",
+  "author": "puppetlabs",
```
Comment (branan) [inline metadata.json:4]: As on Cron, should this be "Puppet" or something  "human readable" like that, raher than just matching our github namespace?

**Comment by tavilarion** [comment]
@melissa @branan can this be merged? It is required for the changelog generation script to work so I can create the pull request for release.

File: metadata.json (hunk 2 above)
Comment (caseywilliams) [reply metadata.json:4]: I'm +1 to "Puppet"

Comment (melissa) [reply metadata.json:4]: I think @binford2k  mentioned in another commit that this has to correspond to the forge user that publishes the module, so it should remain `puppetlabs`. I'm not seeing where that PR went though

Comment (caseywilliams) [reply metadata.json:4]: Ok, sounds good to me, thanks @melissa

**Review by caseywilliams — APPROVED** [review]

**Pull request merged**